test/unit/lib/redmine/menu_manager/menu_helper_test.rb

```diff
@@ -175,20 +175,20 @@ def test_render_menu_node_with_nested_items_and_children
 
     html_node = HTML::Document.new(@response.body)
     assert_select(html_node.root, "li") do
-      assert_select("a.parent-node", "Parent node")
+      assert_select("a.parent-node-menu-item", "Parent node")
       assert_select("ul") do
-        assert_select("li a.child-node", "Child node")
+        assert_select("li a.child-node-menu-item", "Child node")
         assert_select("ul") do
-          assert_select("li a.test-dynamic-child-0", "Test dynamic child 0")
-          assert_select("li a.test-dynamic-child-1", "Test dynamic child 1")
-          assert_select("li a.test-dynamic-child-2", "Test dynamic child 2")
-          assert_select("li a.test-dynamic-child-3", "Test dynamic child 3")
-          assert_select("li a.test-dynamic-child-4", "Test dynamic child 4")
-          assert_select("li a.test-dynamic-child-5", "Test dynamic child 5")
+          assert_select("li a.test-dynamic-child-0-menu-item", "Test dynamic child 0")
+          assert_select("li a.test-dynamic-child-1-menu-item", "Test dynamic child 1")
+          assert_select("li a.test-dynamic-child-2-menu-item", "Test dynamic child 2")
+          assert_select("li a.test-dynamic-child-3-menu-item", "Test dynamic child 3")
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.